pkg/api/init.go

```diff
@@ -0,0 +1,12 @@
+package api
+
+import (
```

Comment:
For decoding of yaml files: https://stackoverflow.com/questions/44306554/how-to-deserialize-kubernetes-yaml-file